.github/workflows/docs-workflow.yaml

```diff
@@ -65,3 +65,116 @@ jobs:
       working-directory: ./docs
       run: |
         git push origin master
+  get-started-docs:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
```suggestion
    needs: [api-docs]
    steps:
```

Reply:
The two workflows can be executed independently, so I would not bind them with a `needs`.